doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -4786,4 +4786,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
+}
```

Comment:
There is no change in this file. Are you able to remove the perceived git changes from this PR? If not let me know and I can help

Reply:
I did notice this, but I had just copy-pasted the contents of this file from the actual repo only so what do I do to fix this?
Like there's literally no change but still git shows it up

Reply:
Fixed it

Reply:
Thank you! Git is challenging sometimes, glad you got it figured out :)